pkg/apiserver/api_installer.go

```diff
@@ -105,7 +106,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 
 	serverVersion := a.group.ServerVersion
 	if len(serverVersion) == 0 {
-		serverVersion = a.group.Version
+		serverVersion = apiutil.GetGroupVersion(a.group.Group, a.group.Version)
```

Comment:
yes, ServerVersion is a groupVersion. I will rename it.

It can be set different from apiutil.GetGroupVersion(a.group.Group, a.group.Version),  so the `if` is needed.